DDDEastAnglia/DataAccess/DefaultSponsorSorter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.DataAccess
+{
+    public static class SponsorExtensions
+    {
+        public static IEnumerable<Sponsor> OrderBySponsorSorter(this IEnumerable<Sponsor> sponsors)
```

Comment:
I don't like this extension method. There is already an `OrderBy` and `OrderByDescending` methods in linq that take a comparer so this extension method feels unnecessary.